packages/deployments/contracts/contracts/interfaces/IConnext.sol

```diff
@@ -251,7 +255,8 @@ interface IConnext {
     uint256 _domain,
     address payable _bridgeRouter,
     address _tokenRegistry, // Nomad token registry
-    address _wrappedNative
+    address _wrappedNative,
+    address payable _promiseRouter
```

Comment:
will this be a payable address, or just an address? (same with `_bridgeRouter`)

Reply:
`_promiseRouter` address should be payable address. because this address should be able to receive ETH as callback fee